### PR TITLE
docs: Add 2023-05-04 meeting notes

### PR DESCRIPTION
**Pull request by sam3k**
Closes https://github.com/eslint/tsc-meetings/issues/427

### CODE REVIEW

File: notes/2023/2023-04-20-transcript.md
Comment (nzakas) [inline notes/2023/2023-04-20-transcript.md:1]: No need to include the transcript manually. We have one autogenerated after meeting notes are posted.

**Review by nzakas — DISMISSED** [review]
LGTM. Would like @mdjermanovic to review before merging.

File: notes/2023/2023-04-20-transcript.md
Comment (sam3k) [reply notes/2023/2023-04-20-transcript.md:1]: Huh, apologies for the delay. I don't recall adding that file. Removed.

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**